policy/modules/admin/portage.fc

```diff
@@ -24,6 +25,12 @@
 /usr/portage/distfiles/svn-src(/.*)?	gen_context(system_u:object_r:portage_srcrepo_t,s0)
 
 /var/db/pkg(/.*)?	gen_context(system_u:object_r:portage_db_t,s0)
+/var/db/repos(/.*)?	gen_context(system_u:object_r:portage_ebuild_t,s0)
+/var/cache/binpkgs(/.*)?  gen_context(system_u:object_r:portage_ebuild_t,s0)
+/var/cache/distfiles/cvs-src(/.*)?	gen_context(system_u:object_r:portage_srcrepo_t,s0)
+/var/cache/distfiles/egit-src(/.*)?	gen_context(system_u:object_r:portage_srcrepo_t,s0)
+/var/cache/distfiles/git-src(/.*)?	gen_context(system_u:object_r:portage_srcrepo_t,s0)
```

Comment:
This should be `git[0-9]-src` like in:

https://gitweb.gentoo.org/proj/hardened-refpolicy.git/tree/policy/modules/admin/portage.fc#n26